x/incentives/keeper/grpc_query_test.go

```diff
@@ -1,13 +1,21 @@
 package keeper_test
 
 import (
+	// "fmt"
 	"time"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
 
+<<<<<<< HEAD
 	"github.com/osmosis-labs/osmosis/v9/x/incentives/types"
 	lockuptypes "github.com/osmosis-labs/osmosis/v9/x/lockup/types"
 	pooltypes "github.com/osmosis-labs/osmosis/v9/x/pool-incentives/types"
+=======
+	query "github.com/cosmos/cosmos-sdk/types/query"
+	"github.com/osmosis-labs/osmosis/v7/x/incentives/types"
+	lockuptypes "github.com/osmosis-labs/osmosis/v7/x/lockup/types"
+	pooltypes "github.com/osmosis-labs/osmosis/v7/x/pool-incentives/types"
+>>>>>>> b464728 (Fix incentives query filtering (#1759))
```

Comment:
Hopefully we make automation in #1772 to not have to long term do this
```suggestion
	query "github.com/cosmos/cosmos-sdk/types/query"
	"github.com/osmosis-labs/osmosis/v9/x/incentives/types"
	lockuptypes "github.com/osmosis-labs/osmosis/v9/x/lockup/types"
	pooltypes "github.com/osmosis-labs/osmosis/v9/x/pool-incentives/types"
>>>>>>> b464728 (Fix incentives query filtering (#1759))
```